tests/lib/snaps/test-snapd-sh-core20/meta/snap.yaml

```diff
@@ -0,0 +1,8 @@
+name: test-snapd-sh-core20
```

Comment:
what is this snap used for in this PR?

Reply:
it is used in the basic and snapd-refresh snaps. If we install test-snapd-sh it requires core, and the idea it that test-snapd-sh-core20 it does not require it.